tests/test_cli_download_files.py

```diff
@@ -31,53 +32,92 @@ def test_download_files():
 
 def test_download_files_filter_name():
     """Test download_files() command with filter-name options."""
-    test_file = "3005/0d0714743f0204ed3c0144941e6ce248.configFile.py"
+    test_file = "5500/BuildFile.xml"
     if os.path.isfile(test_file):
         os.remove(test_file)
     test_download_files_filter = CliRunner()
-    test_result_name = test_download_files_filter.invoke(
+    test_result_name_one = test_download_files_filter.invoke(
         download_files,
         [
             "--recid",
-            3005,
+            5500,
             "--filter-name",
-            "0d0714743f0204ed3c0144941e6ce248.configFile.py",
+            "BuildFile.xml",
         ],
     )
-    assert test_result_name.exit_code == 0
+    assert test_result_name_one.exit_code == 0
     assert os.path.isfile(test_file) is True
-    assert test_result_name.output.endswith("\n==> Success!\n")
+    assert test_result_name_one.output.endswith("\n==> Success!\n")
     if os.path.isfile(test_file):
         os.remove(test_file)
+    test_result_name_two = test_download_files_filter.invoke(
+        download_files, ["--recid", 5500, "--filter-name", "test_name.py"]
+    )
+    assert test_result_name_two.exit_code == 1
+    assert test_result_name_two.output == "\nNo files matching the filters\n"
 
 
 def test_download_files_filter_regexp():
     """Test download_files() command with filter-regexp options."""
-    test_file = "3005/0d0714743f0204ed3c0144941e6ce248.configFile.py"
-    if os.path.isfile(test_file):
-        os.remove(test_file)
+    test_one_file = "3005/0d0714743f0204ed3c0144941e6ce248.configFile.py"
+    test_two_files = [
+        "5500/HiggsDemoAnalyzer.cc",
+        "5500/M4Lnormdatall.cc",
+        "5500/M4Lnormdatall_lvl3.cc",
+    ]
+    if os.path.isfile(test_one_file):
+        os.remove(test_one_file)
     test_download_files_filter = CliRunner()
-    test_result_regexp = test_download_files_filter.invoke(
+    test_result_regexp_one = test_download_files_filter.invoke(
         download_files, ["--recid", 3005, "--filter-regexp", "py$"]
     )
-    assert test_result_regexp.exit_code == 0
-    assert os.path.isfile(test_file) is True
-    assert test_result_regexp.output.endswith("\n==> Success!\n")
-    if os.path.isfile(test_file):
-        os.remove(test_file)
+    assert test_result_regexp_one.exit_code == 0
+    assert test_result_regexp_one.output.endswith("\n==> Success!\n")
+    if os.path.isfile(test_one_file):
+        os.remove(test_one_file)
+    else:
+        pytest.fail("{} not downloaded".format(test_one_file))
+
+    for file_location in test_two_files:
+        if os.path.isfile(file_location):
+            os.remove(file_location)
+    test_result_regexp_two = test_download_files_filter.invoke(
+        download_files, ["--recid", 5500, "--filter-regexp", "\.cc$"]
```

Comment:
I get one pytest warning on this:

```
tests/test_cli_download_files.py:85
  /home/simko/private/project/opendata/src/cernopendata-client/tests/test_cli_download_files.py:85: DeprecationWarning: invalid escape sequence \.
    download_files, ["--recid", 5500, "--filter-regexp", "\.cc$"]

...

16 passed, 1 warning in 7.67s
```

Can you please fix? Would be nice if all tests are reported as passing.

Reply:
It is because of `\` in the regexp!